tekton/ci/interceptors/github/pkg/proto/v1alpha1/config.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
```

Comment:
Just curious on the decision to use proto for the config vs just a go struct?

Reply:
Primarily a shortcut to codegen structs with getters, json tags, and other common things I didn't want to do by hand. 😅

Even though we're not really using this message for RPCs, it makes it easier to ensure consistent field behavior if/when this changes.